data_model/derive/src/lib.rs

```diff
@@ -92,10 +93,10 @@ mod id;
 /// }
 ///
 /// ```
+#[proc_macro_error::proc_macro_error]
 #[proc_macro_derive(IdOrdEqHash, attributes(id))]
 pub fn id_derive(input: TokenStream) -> TokenStream {
```

Comment:
Would appreciate if you updated the docs here to reflect changes in attribute usage.

Reply:
done